### PR TITLE
Add feedback form to explore page

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
nice

File: static/js/apps/explore/app.tsx [hunk 1]
```diff
@@ -68,8 +71,9 @@ export function App(): JSX.Element {
   const [hashParams, setHashParams] = useState<ParsedQuery<string>>(
     queryString.parse(window.location.hash)
   );
-  const [savedContext, setSavedContext] = useState<any>({});
+  const [savedContext, setSavedContext] = useState<any>([]);
```
Comment (pradh) [inline static/js/apps/explore/app.tsx:74]: Change line 119 on RHS to [] too?

File: static/js/apps/explore/app.tsx [hunk 2]
```diff
@@ -31,13 +31,16 @@ import { SVG_CHART_HEIGHT } from "../../constants/app/nl_interface_constants";
...
+import { getFeedbackLink } from "../../utils/nl_interface_utils";
 import { updateHash } from "../../utils/url_utils";
 import { ParentPlace } from "./parent_breadcrumbs";
 import { Sidebar } from "./sidebar";
 
 const PAGE_ID = "explore";
 const DEFAULT_PLACE = "geoId/06";
 const DEFAULT_TOPIC = "dc/topic/Root";
+const FEEDBACK_LINK =
+  "https://docs.google.com/forms/d/14oXA39Il7f20Rvtqkx_KZNn2NXTi7D_ag_hiX8oH2vc/viewform?usp=pp_url";
```
Comment (pradh) [inline static/js/apps/explore/app.tsx:43]: nit: give access to datcom-core?

**Review by pradh — APPROVED** [review]
Thanks Bo!

**Pull request merged**